src/compiler/crystal/config.cr

```diff
@@ -8,12 +8,16 @@ module Crystal
       version_and_sha.first
     end
 
+    def self.llvm_version
+      LibLLVM::VERSION
+    end
+
     def self.description
       version, sha = version_and_sha
       if sha
-        "Crystal #{version} [#{sha}] (#{date})"
+        "Crystal #{version} [#{sha}] (#{date}) LLVM #{llvm_version}"
       else
-        "Crystal #{version} (#{date})"
+        "Crystal #{version} (#{date}) LLVM #{llvm_version}"
```

Comment:
I thought about that - a little bit inspired in `docker version` - but I couldn't come with any other thing I wanted to see reported there.

I thought maybe a Crystal program would specify this versions in it's own `--version` command - say, running `my-tool --version` would report both `my-tool`'s, Crystal's & LLVM's versions, but @asterite somewhat convinced me you shouldn't care about LLVM version in client code (btw, do you agree with this?).

I'm open to change my mind, anyway :)